motoko/invoice-canister/examples/motoko-seller-client/canister_ids.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
canister_ids.json shouldn't be added to source control, so that others can clone and deploy the project and get their own canisters.